openml/extensions/sklearn/extension.py

```diff
@@ -290,13 +301,15 @@ def _deserialize_sklearn(
                     rval = self._deserialize_function(value)
                 elif serialized_type == 'component_reference':
                     assert components is not None  # Necessary for mypy
-                    value = self._deserialize_sklearn(value, recursion_depth=depth_pp)
+                    value = self._deserialize_sklearn(value, recursion_depth=depth_pp,
+                                                      strict_version=strict_version)
                     step_name = value['step_name']
                     key = value['key']
                     component = self._deserialize_sklearn(
                         components[key],
                         initialize_with_defaults=initialize_with_defaults,
-                        recursion_depth=depth_pp
+                        recursion_depth=depth_pp,
+                        strict_version=strict_version
```

Comment:
Here as well.

Reply:
Do you have any experience with black? Maybe we could use that to enforce such rather uncommon (and so far) undocumented styles.

Reply:
I don't sorry. scikit-learn-extra uses it.

Reply:
I opened an issue for this.